zipline/pipeline/domain.py

```diff
@@ -184,26 +184,38 @@ def __repr__(self):
         )
 
 
+AT_EQUITIES = EquityCalendarDomain(CountryCode.AUSTRIA, 'XWBO')
+AU_EQUITIES = EquityCalendarDomain(CountryCode.AUSTRALIA, 'XASX')
 BE_EQUITIES = EquityCalendarDomain(CountryCode.BELGIUM, 'XBRU')
 CA_EQUITIES = EquityCalendarDomain(CountryCode.CANADA, 'XTSE')
 CH_EQUITIES = EquityCalendarDomain(CountryCode.SWITZERLAND, 'XSWX')
 DE_EQUITIES = EquityCalendarDomain(CountryCode.GERMANY, 'XFRA')
+ES_EQUITIES = EquityCalendarDomain(CountryCode.SPAIN, 'XMAD')
 FR_EQUITIES = EquityCalendarDomain(CountryCode.FRANCE, 'XPAR')
 GB_EQUITIES = EquityCalendarDomain(CountryCode.UNITED_KINGDOM, 'XLON')
+HK_EQUITIES = EquityCalendarDomain(CountryCode.HONG_KONG, 'XHKG')
+IT_EQUITIES = EquityCalendarDomain(CountryCode.ITALY, 'XMIL')
 JP_EQUITIES = EquityCalendarDomain(CountryCode.JAPAN, 'XTKS')
 NL_EQUITIES = EquityCalendarDomain(CountryCode.NETHERLANDS, 'XAMS')
+NZ_EQUITIES = EquityCalendarDomain(CountryCode.NEW_ZEALAND, 'XNZE')
 PT_EQUITIES = EquityCalendarDomain(CountryCode.PORTUGAL, 'XLIS')
 US_EQUITIES = EquityCalendarDomain(CountryCode.UNITED_STATES, 'XNYS')
 
 BUILT_IN_DOMAINS = [
+    AT_EQUITIES,
```

Comment:
I think you'll need to update `test_built_in_equity_calendar_domain_defaults` in `test_domain.py`, and possibly some other stuff in that test suite

Reply:
Thanks